lib/screen/workout/workout_list.dart

```diff
@@ -0,0 +1,198 @@
+import 'package:event_bloc/event_bloc.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:simple_exercise_builder/bloc/workout/workout.dart';
+import 'package:simple_exercise_builder/model/workout.dart';
+import 'package:simple_exercise_builder/screen/workout/search_bar.dart';
+import 'package:simple_exercise_builder/screen/workout/workout_category_list.dart';
+
+class WorkoutList extends StatefulWidget {
```

Comment:
Please add Screen (WorkoutListScreen) to this name. trying to differentiate between this and [WorkoutCategoryList] should be easy. Also, the name collides so much which makes it hard to code review. So just add screen.

Reply:
Renamed at 3b04b6f